sympy/polys/domains/domain.py

```diff
@@ -1287,6 +1287,10 @@ def half_gcdex(self, a, b):
         s, t, h = self.gcdex(a, b)
         return s, h
 
+    def gcdex_steps(self, a, b):
+        """Intermediate steps of the extended GCD algorithm. """
+        raise NotImplementedError
```

Comment:
This should be removed because it is a function now rather than a domain method.